test/org/sagebionetworks/bridge/services/AccountWorkflowServiceTest.java

```diff
@@ -273,7 +271,7 @@ public void sendPhoneVerificationTokenNoPhone() {
     @Test
     public void sendPhoneVerificationTokenThrottled() {
         // Throttle limit is 2. Make 3 requests, and send only 2 emails.
-        when(service.getNextToken()).thenReturn(TOKEN);
+     // */when(service.getNextToken()).thenReturn(TOKEN);
```

Comment:
This fixes the test but since it's testing throttling, and not the content of the sms message provider, I don't think it needs to be fixed (but I do need to remove the commented out code).